TypeCobol/Compiler/CodeModel/SymbolTable.cs

```diff
@@ -734,10 +734,25 @@ internal void AddParagraph(Paragraph paragraph)
             Add(Paragraphs, paragraph);
         }
 
-        public IList<Paragraph> GetParagraph(string name)
+        public IList<Paragraph> GetParagraph(SymbolReference parRef)
         {
-            Paragraphs.TryGetValue(name, out var values);
-            if (values != null) return values.ToList();  //.ToList so the caller cannot modify our stored list
+            //Retrieve all paragraphs having the same name as parRef
+            if (Paragraphs.TryGetValue(parRef.NameLiteral.Value, out var values))
```

Comment:
`parRef.Name`

Reply:
parRef.NameLiteral.Value holds the value needed for the test as parRef.Name is actually the qualified name.

Reply:
You're right `QualifiedSymbolReference` overrides the `Name` property.